lib/logstash/filters/elapsed.rb

```diff
@@ -109,6 +109,12 @@ class LogStash::Filters::Elapsed < LogStash::Filters::Base
   # to the "end event"; if it's set to `true` a new "match event" is created.
   config :new_event_on_match, :validate => :boolean, :required => false, :default => false
 
+  # This property manage what to do when several "start events" were received
+  # before an "end event" for a concrete ID. If it's set to `true` (default
+  # value), the first "start event" will be used; if it's set to `false`,
+  # the last "start event" will be used.
+  config :keep_first_start_event, :validate => :boolean, :required => false, :default => true
```

Comment:
I'd rather make the two possible behaviours of either keeping the first or the last start event by making this property a string:

```ruby
config :keep_start_event, :validate => ['first', 'last'], :required => false, :default => 'first'`
```

Reply:
Also, please also document this option in the `docs/index.asciidoc` file